inlong-dashboard/src/locales/cn.json

```diff
@@ -233,6 +233,8 @@
   "meta.Group.AccessPeakPerSecond": "每秒接入峰值",
   "meta.Group.Stripe/Second": "条/秒",
   "meta.Group.SingleStripMaximumLength": "单条最大长度",
+  "meta.Group.Partition": "分区数",
```

Comment:
The key use `meta.Group.Kafka.Partition`. Others similar to the same need to be modified.